generators/client-2/templates/src/main/webapp/app/layouts/_index.ts

```diff
@@ -2,7 +2,8 @@ export * from './error/error.component';
 export * from './error/error.state';
 export * from './footer/footer.component';
 export * from './navbar/navbar.component';
-export * from './navbar/active-menu.directive';
+<%_ if (enableTranslation){ _%>export * from './navbar/active-menu.directive';
```

Comment:
The export statement should be in a new line. The templating with  _ automatically trims space and lines